braid/_braid.h

```diff
@@ -320,6 +347,16 @@ typedef struct _braid_Core_struct
    braid_Real             localtime;        /**< local wall time for braid_Drive() */
    braid_Real             globaltime;       /**< global wall time for braid_Drive() */
 
+   /** Delta correction and Lyapunov vector estimation*/
+   braid_Int              delta_correct;    /**< turns on Delta correction to potentially accelerate convergence */
+   braid_Int              delta_rank;       /**< for low rank Delta correction */
+   braid_Int              delta_defer_lvl;  /**< Delta correction will be turned off until this coarse level */
```

Comment:
Does delta_defer_lvl  only run delta correction on the coarsest k-levels, or is it the other way around, delta correction is only run on the finest k-levels?